docs/topics/db/aggregation.txt

```diff
@@ -425,6 +425,27 @@ publisher's that have at least one book with a rating exceeding 3.0. The second
 query asks for the average of a publisher's book's ratings for only those
 ratings exceeding 3.0.
 
+Using multivalued filtering against a previously annotated aggregation might
+still yield duplicate results.
+
+Given:
+
+* Publisher A has two books with each having 100 pages.
+
+Here's an example demonstrating this behavior::
+
+    >>> a = Publisher.objects.annotate(total_pages(Sum("books__pages"))
+    >>> a.total_pages
+    200
+    >>> b = Publisher.objects.annotate(Sum("books__pages")).filter(books__in=Books.objects.all())
+    >>> b.total_pages
+    400
+
+This is behavior is caused because ``filter()`` calls won't reuse multivalued
```

Comment:
IIUC, the point from @charettes is that we should link explicitly to the "Spanning multi-valued relationships" section (which already has a reference target named `spanning-multi-valued-relationships`).